### PR TITLE
FHAC-565: NetworkAccessPointTypeCode fix

### PR DESCRIPTION
**Pull request by christophermay07**
- Unit test refactoring
- NetworkAccessPointTypeCode fix (setting appropriate value for IPs)

### CODE REVIEW

File: Product/Production/Services/AuditRepositoryCore/src/test/java/gov/hhs/fha/nhinc/audit/transform/AuditTransformsTest.java [hunk 1]
```diff
@@ -78,38 +86,32 @@ protected void testGetEventIdentificationType(LogEventRequestType request, Strin
         Boolean isRequesting) {
 
         EventIdentificationType eventIdentificationType = request.getAuditMessage().getEventIdentification();
+
         if (isRequesting) {
             assertEquals(getAuditTransforms().getServiceEventActionCodeRequestor(),
                 eventIdentificationType.getEventActionCode());
```
Comment (alameluchidambaram) [inline Product/Production/Services/AuditRepositoryCore/src/test/java/gov/hhs/fha/nhinc/audit/transform/AuditTransformsTest.java:92]: Can you add error message to be thrown on these assertions to be consistent?

File: Product/Production/Services/AuditRepositoryCore/src/test/java/gov/hhs/fha/nhinc/audit/transform/AuditTransformsTest.java [hunk 2]
```diff
@@ -78,38 +86,32 @@ protected void testGetEventIdentificationType(LogEventRequestType request, Strin
...
-        assertEquals(getAuditTransforms().getServiceEventCodeSystem(),
-            eventIdentificationType.getEventID().getCodeSystemName());
-
-        if (isRequesting) {
             assertEquals(getAuditTransforms().getServiceEventDisplayRequestor(), eventIdentificationType.getEventID()
                 .getDisplayName());
         } else {
+            assertEquals(getAuditTransforms().getServiceEventActionCodeResponder(),
+                eventIdentificationType.getEventActionCode());
+            assertEquals(getAuditTransforms().getServiceEventIdCodeResponder(),
```
Comment (alameluchidambaram) [inline Product/Production/Services/AuditRepositoryCore/src/test/java/gov/hhs/fha/nhinc/audit/transform/AuditTransformsTest.java:100]: same comment as above.

**Comment by jsmith2** [comment]
Approving, only comment is that I'm curious what all the updated unit tests coverage is.  Will merge after Alamelu's comments are addressed.

**Comment by christophermay07** [comment]
Closing PR due to irreconcilable differences; will open a new PR later.

**Pull request closed without merging**